torch/_functorch/aot_autograd.py

```diff
@@ -3620,7 +3660,8 @@ def aot_dispatch_autograd(flat_fn, flat_args: List[Any], aot_config: AOTConfig,
 
             forward_saved_for_backwards_strides = None
             if fwd_output_strides is not None:
-                forward_saved_for_backwards_strides = fwd_output_strides[fw_metadata.tensors_saved_for_backwards_slice]
+                inner_meta.num_symints_saved_for_bw = fw_metadata.num_symints_saved_for_bw
```

Comment:
nit: I would move this up to the same place where `fw_metadata.mum_symints_saved_for_bw` is also set, for clarity.